src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -266,6 +266,18 @@ public abstract partial class TransportContext
         protected TransportContext() { }
         public abstract System.Security.Authentication.ExtendedProtection.ChannelBinding GetChannelBinding(System.Security.Authentication.ExtendedProtection.ChannelBindingKind kind);
     }
+
+    public static class HttpVersion
```

Comment:
Please match what's defined in the standard repo. If that gets updated, then this can, too, but for now keep it as what's defined there.